lib/config/proxy.go

```diff
@@ -45,17 +45,25 @@ type Metrics struct {
 }
 
 type KeepAlive struct {
-	Enabled bool          `yaml:"enabled,omitempty" toml:"enabled,omitempty" json:"enabled,omitempty"`
-	Idle    time.Duration `yaml:"idle,omitempty" toml:"idle,omitempty" json:"idle,omitempty"`
-	Cnt     int           `yaml:"cnt,omitempty" toml:"cnt,omitempty" json:"cnt,omitempty"`
-	Intvl   time.Duration `yaml:"intvl,omitempty" toml:"intvl,omitempty" json:"intvl,omitempty"`
+	Enabled bool `yaml:"enabled,omitempty" toml:"enabled,omitempty" json:"enabled,omitempty"`
+	// Idle, Cnt, and Intvl works only when the connection is idle. User packets will interrupt keep-alive.
+	// If the peer crashes and doesn't send FIN, the connection will be closed within Idle+Cnt*Intvl.
```

Comment:
It is actually waiting for `ACK`.

Reply:
I mean, when it's idle. The peer should send FIN before it closes. But if it crashes, it doesn't send FIN.

Reply:
I knew. But if it is not crashed, it will also be disconnected. For example, delayed ack feature.